### PR TITLE
REL-3438: Increase OT local DB storage limit.

### PR DESCRIPTION
**Pull request by sraaphorst**
This increases the limit on the local OT storage from 50 MB to 100 MB as requested.

### CODE REVIEW

File: bundle/jsky.app.ot/src/main/scala/jsky/app/ot/viewer/open/OpenDialog.scala [hunk 1]
```diff
@@ -115,7 +115,7 @@ class OpenDialog private(db: IDBDatabaseService, auth: KeyChain, vcs: VcsRegistr
   var selection: List[ISPProgram] = Nil
   val filter = new DBProgramChooserFilter(DBProgramChooserFilter.Mode.localAndRemote)
 
-  val MaxM = 50
+  val MaxM = 100
```
Comment (cquiroz) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/viewer/open/OpenDialog.scala]: This seems just for the UI? where is the size enforced?

Comment (sraaphorst) [reply bundle/jsky.app.ot/src/main/scala/jsky/app/ot/viewer/open/OpenDialog.scala]: If you look at L367 of `OpenDialog`, you can see that the limit is enforced and programs are only opened if there is enough room in `MaxK`, which is calculated from `MaxM`.

Comment (sraaphorst) [reply bundle/jsky.app.ot/src/main/scala/jsky/app/ot/viewer/open/OpenDialog.scala]: Ooops. I changed the version. Grrrr. This perpetual 2016A versioning is annoying.

Comment (cquiroz) [reply bundle/jsky.app.ot/src/main/scala/jsky/app/ot/viewer/open/OpenDialog.scala]: Ok, good to have UI and side effects mixed 😄

Comment (swalker2m) [reply bundle/jsky.app.ot/src/main/scala/jsky/app/ot/viewer/open/OpenDialog.scala]: How so? The UI shows and enforces a limit on how many programs you can have locally. It’s related to keeping the used heap memory down to a reasonable size but it isn’t the same thing as the command line max heap parameter.

Comment (swalker2m) [reply bundle/jsky.app.ot/src/main/scala/jsky/app/ot/viewer/open/OpenDialog.scala]: Btw, the “size” isn’t even heap memory per se but the size of the sum of the corresponding serialized program files.

**Pull request merged**